packages/flutter/lib/src/cupertino/button.dart

```diff
@@ -139,9 +220,14 @@ class CupertinoButton extends StatefulWidget {
 
   /// The radius of the button's corners when it has a background color.
   ///
-  /// Defaults to round corners of 8 logical pixels.
+  /// Defaults to [_kCupertinoButtonSizeBorderRadius], based on [size].
```

Comment:
Dart doc doesn't pick up bracket references to private references well. No issue if we make `_kCupertinoButtonSizeBorderRadius` public.